### PR TITLE
Create an endpoint for quick answer game. Modified database model.

### LINKED ISSUE #24: Create endpoint for quick game page.
**Is your feature request related to a problem? Please describe.**
Two endpoints.
One returns all data of one game.
That data form is :
```
{problems:[{content_id:..., japanese_slang}],
    deal:[[slang1, slang2, slang2], [slang1, slang2, slang3], ...]}
```
The other receives request for game answer.
User can send answer to the endpoint and it returns correct answer or not.

**Describe the solution you'd like**
Django rest api.

### PR DESCRIPTION
**Pull request by ebisuG**

### CODE REVIEW

File: backend/api/models.py [hunk 1]
```diff
@@ -37,6 +37,7 @@ def __str__(self):
 class Content(models.Model):
     content_id = models.AutoField(primary_key=True)
     japanese_slang = models.CharField(max_length=255)
+    english_slang = models.CharField(max_length=255)
```
Comment (FilippoQuattrocchi) [inline backend/api/models.py:40]: Maybe is better a textfield because can contain long string

**Review by FilippoQuattrocchi — CHANGES_REQUESTED** [review]
The deploy is failed, please check

File: backend/api/models.py (hunk 1 above)
Comment (ebisuG) [reply backend/api/models.py:40]: @FilippoQuattrocchi 
I extended each field to 1024, not only english_slang.

File: backend/api/models.py
Comment (FilippoQuattrocchi) [inline backend/api/models.py:1]: @ebisuG  i think we need 1024 only for the descrption the others we can keep 255

Comment (ebisuG) [reply backend/api/models.py:1]: @FilippoQuattrocchi 
Sure! I changed and created new migrations. Maybe we should merge our migrations to one file later.

Comment (FilippoQuattrocchi) [reply backend/api/models.py:1]: we can do with squash migrations

File: backend/api/models.py [hunk 2]
```diff
@@ -36,9 +36,9 @@ def __str__(self):
 
 class Content(models.Model):
     content_id = models.AutoField(primary_key=True)
-    japanese_slang = models.CharField(max_length=1024)
+    japanese_slang = models.CharField(max_length=255)
     english_slang = models.CharField(max_length=1024)
```
Comment (FilippoQuattrocchi) [inline backend/api/models.py]: English slang too!

**Review by FilippoQuattrocchi — APPROVED** [review]

**Pull request merged**